release.yaml

```diff
@@ -67,7 +67,9 @@ rancher-gke-operator-crd:
 - 100.0.2+up1.1.3-rc1
 rancher-cis-benchmark:
 - 2.0.3-rc1
```

Comment:
need to remove these old charts! Can you add a `make remove` commit before you run `make charts`?

`CHART=rancher-cis-benchmark VERSION=2.0.3-rc1 make remove && CHART=rancher-cis-benchmark-crd VERSION=2.0.3-rc1 make remove`